packages/openapi3/src/xml-module.ts

```diff
@@ -0,0 +1,202 @@
+import {
+  ArrayModelType,
+  IntrinsicScalarName,
+  Model,
+  ModelProperty,
+  Program,
+  Scalar,
+  isArrayModelType,
+  resolveEncodedName,
+} from "@typespec/compiler";
+import { ArrayBuilder, ObjectBuilder } from "@typespec/compiler/emitter-framework";
+import { getNs, isAttribute, isUnwrapped } from "@typespec/xml";
+import { reportDiagnostic } from "./lib.js";
+import { ResolvedOpenAPI3EmitterOptions } from "./openapi.js";
+import { getSchemaForStdScalars } from "./std-scalar-schemas.js";
+import { OpenAPI3Schema, OpenAPI3XmlSchema } from "./types.js";
+
+const B = {
```

Comment:
I think this was an old thing that is uncesssary, just call `new ArrayBuilder(v)`